docs/resources/core/internet_gateway.md

```diff
@@ -1,6 +1,10 @@
 # oci\_core\_internet\_gateways
 
-Gets a list of internet gateways.
+[InternetGateway Reference][0162d0a8]
+
+  [0162d0a8]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/InternetGateway/ "InternetGatewayReference"
+
+Gets a list of Internet Gateways.
```

Comment:
I know that isn't part of your change, but any resource doc that says "Gets a list of" is wrong - probably copied from the data source.